### PR TITLE
Refresh CLI output by input.

### PR DESCRIPTION
**Pull request by tushuhei**
Change-Id: I3257d418e3af3d668d17d2c16e1bd7ec9d2b01ea

### CODE REVIEW

File: mozc-nazoru/src/nazoru/keyboard_recorder.py [hunk 1]
```diff
@@ -260,6 +263,8 @@ def record(self):
         start_time = now
       elapsed_ms = int((now - start_time).total_seconds() * 1000)
       data.append((self.KEYS[event.code], elapsed_ms))
+      clear_screen()
+      sys.stdout.write('%s\r' % (' '.join(datum[0] for datum in data)))
```
Comment (makotoshimazu) [inline mozc-nazoru/src/nazoru/keyboard_recorder.py]: Wdyt of using print() here?
I think it'll keep the final result in the screen and it'd be helpful to show what's happening on RPi.

File: mozc-nazoru/src/nazoru/keyboard_recorder.py [hunk 2]
```diff
@@ -260,6 +263,8 @@ def record(self):
         start_time = now
       elapsed_ms = int((now - start_time).total_seconds() * 1000)
       data.append((self.KEYS[event.code], elapsed_ms))
+      clear_screen()
```
Comment (makotoshimazu) [inline mozc-nazoru/src/nazoru/keyboard_recorder.py]: Is it okay to remove clear_screen() here if we change the following line to print()?

**Comment by tushuhei** [comment]
Thank you for suggestion! I don't have a device right now, so I will check how it works once I'm back.

**Comment by tushuhei** [comment]
Let me merge this for now and work on your suggestions in the next round.

**Pull request merged**